### DISCUSSION TITLE
failed to process outbound traffic > proxy/freedom: connection ends > context canceled

### DISCUSSION THREAD

**Discussion by mobinseven** [v2fly/v2ray-core#2746] [Q&A]
Hello

I am getting this message on both server and client:

`app/proxyman/outbound: failed to process outbound traffic > proxy/freedom: connection ends > context canceled`

On some of transactions, not all of them. 

This is my server config:

```
{
  "log": {
    "loglevel": "info"
  },
  "inbounds": [
    {
      "port": 10086,
      "protocol": "vmess",
      "settings": {
        "clients": [
          {
            "id": "UUID",
            "alterId": 0
          }
        ]
      }
    }
  ],
  "outbounds": [
    {
      "protocol": "freedom",
      "settings": {}
    }
  ]
}
```

And this is my client config:

```
{
  "inbounds": [{
    "port": 1080,
    "listen": "127.0.0.1",
    "protocol": "socks",
    "settings": {
      "udp": true
    }
  }],
  "outbounds": [{
    "protocol": "vmess",
    "settings": {
      "vnext": [{
        "address": "server",
        "port": 10086,
        "users": [{ "id": "UUID" }]
      }]
    }
  },{
    "protocol": "freedom",
    "tag": "direct",
    "settings": {}
  }],
  "routing": {
    "domainStrategy": "IPOnDemand",
    "rules": [{
      "type": "field",
      "ip": ["geoip:private"],
      "outboundTag": "direct"
    }]
  }
}
```

The connection was working properly for months without this issue. Nothing has been changed. Again, not all transactions are affected. Just some of them fail.

I have read these discussions:

- https://github.com/v2fly/v2ray-core/discussions/1442
- https://github.com/v2fly/v2ray-core/discussions/1508

There is no problem with my 'alterId' or system clock.

**Comment by sci-42ver** [comment]
IMHO, this message can be ignored if the proxy is well.

On my machine, these messages appear many times
```bash
2023/11/12 14:28:04 [Info] [23456180] app/proxyman/inbound: connection ends > proxy/http: connection ends > context canceled                                                         
2023/11/12 14:28:04 [Info] [23456180] app/proxyman/outbound: failed to process outbound traffic > proxy/freedom: connection ends > context canceled
```
but the proxy still works.

**Reply by mobinseven** [reply]
I was not receiving this message before. Plus, the quality of connection is too low, as of the beginning of this issue.

Btw, thanks for your attention. It seems no one cares too much here.

**Comment by sudharadev** [comment]


**Reply by mobinseven** [reply]
No. I quitted using V2ray.
